news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,285 @@
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/VIDEO_ID?start=XXX&end=XXX" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The last losers' bracket match for Semifinals between Hong Kong (#6) and Peru (#8) had the audience talking about a potential tiebreaker even before the match started, considering the close seed gap between the two teams. With both of them having their respective longest run in 7K MWC history, the face-off determined which team would get eliminated from there on out while the other survives to continue another world cup week in Finals.
+
+Hong Kong protecting [REDALiCE feat. Ayumi Nomiya - Little Star (Full ver.)](https://osu.ppy.sh/beatmapsets/2317896#mania/4964062) (RC2) while banning [Flying Steps - We Are Electric](https://osu.ppy.sh/beatmapsets/2317884#mania/4964044) (RC3) was a rather unexpected decision according to the audience despite being a very common strategy in MWC matches in general. Meanwhile, Peru protected [SiLiS - Hebe](https://osu.ppy.sh/beatmapsets/2317981#mania/4964313) (RC5) and banned [Elliot Hsu - A Guiding Star](https://osu.ppy.sh/beatmapsets/2317897#mania/4964065) (LN1), which was a smart call given that Hong Kong did get a 125,290-point lead against Germany on that map.
+
+Hong Kong opened up the match with their own protect, but it didn't work out as intended as Peru proceeded to get the first break point of the match. With Peru winning their subsequent pick and with the scoreboard being at 0-2 in favor of Peru, Hong Kong tried to pick [seatrus & Halv - The Core](https://osu.ppy.sh/beatmapsets/2317878#mania/4964035) (LN2) thinking it would be the best pick, considering Peru banned LN1. However, the point streak would continue on, tallying the score up to 0-4 in favor of Peru who was on the verge of a match point after winning their protect.
+
+Hong Kong's 3rd pick, [Lime - Pixel Planet](https://osu.ppy.sh/beatmapsets/2317895#mania/4964061) (RC6) made things slightly more interesting as ::{ flag=PE }:: [-luvv](https://osu.ppy.sh/users/23731514) started off with the wrong foot and gave their rivals a lead hard to come back from. The same can be said about Peru's next pick on [linear ring - enchanted love](https://osu.ppy.sh/beatmapsets/2317898#mania/4964066) (HB2), where they got a team-wide combo reset in the middle of the map, giving Hong Kong their second consecutive point and their only break point of the match.
+
+While the audience was very thrilled for a potential comeback from the Hongkongers, this would unfortunately not be possible after losing their pick on [Xyris - FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1) to a close lead of 6,758 points giving the match point to Peru. Said team then closed out the match by winning their own pick on [The Flashbulb - Six Acid Strings](https://osu.ppy.sh/beatmapsets/2318102#mania/4964713) (RC4), scoring a final tally of 2-6 in favor of Peru. 
```

Comment:
```suggestion
While the audience was very thrilled for a potential comeback from the Hongkongers, this would unfortunately not be possible after losing their pick on [Xyris - FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1) to a close lead of 6,758 points giving the match point to Peru. Said team then closed out the match by winning their own pick on [The Flashbulb - Six Acid Strings](https://osu.ppy.sh/beatmapsets/2318102#mania/4964713) (RC4), scoring a final tally of 2–6 in favor of Peru.
```

Reply:
Britishified

```suggestion
While the audience was very thrilled for a potential comeback from the Hongkongers, this would unfortunately not be possible after losing their pick on [Xyris - FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1) to a close lead of 6,758 points giving the match point to Peru. Said team then closed out the match by winning their own pick on [The Flashbulb - Six Acid Strings](https://osu.ppy.sh/beatmapsets/2318102#mania/4964713) (RC4), scoring a final tally of 2–6 in favour of Peru.
```